Packs/Phishing/Scripts/CheckEmailAuthenticity/CheckEmailAuthenticity.py

```diff
@@ -109,15 +109,40 @@ def auth_check(spf_data, dkim_data, dmarc_data, override_dict):
     return 'Pass'
 
 
+def get_authentication_value(headers, original_authentication_header):
+    """
+    Handel the case where the authentication header is given under a different header.
+    This header is represented by the 'original_authentication_header' argument.  This can happen when there is an
+     intermediate server which changes the email and holds the original value of the header in a different header.
+     For more info, see issue #46364.
```

Comment:
```suggestion
    Handles the case where the authentication header is given under a different header.
    This header is represented by the 'original_authentication_header' argument.  This can happen when an intermediate server changes the email and holds the original value of the header in a different header.
    For more info, see issue #46364.
```